x-pack/plugins/synthetics/server/legacy_uptime/lib/saved_objects/migrations/monitors/8.8.0.ts

```diff
@@ -118,3 +126,73 @@ const omitZipUrlFields = (fields: BrowserFields) => {
 
   return formatSecrets(validationResult.decodedMonitor);
 };
+
+const updateThrottlingFields = (
+  doc: SavedObjectUnsanitizedDoc<
+    SyntheticsMonitorWithSecrets &
+      Partial<{
+        [LegacyConfigKey.THROTTLING_CONFIG]: string;
+        [LegacyConfigKey.IS_THROTTLING_ENABLED]: boolean;
+        [LegacyConfigKey.DOWNLOAD_SPEED]: string;
+        [LegacyConfigKey.UPLOAD_SPEED]: string;
+        [LegacyConfigKey.LATENCY]: string;
+      }>
+  >
+) => {
+  const { attributes } = doc;
+
+  const migrated = {
+    ...doc,
+    attributes: {
+      ...doc.attributes,
+      [ConfigKey.CONFIG_HASH]: '',
```

Comment:
I don't think you need this since it was already applied to every monitor in the previous migration steps, but it's not hurting anything I suppose.